core/lib/dal/migrations/20231212121822_add_miniblocks_fee_account_address.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE miniblocks
+    ADD COLUMN fee_account_address BYTEA NOT NULL DEFAULT '\x0000000000000000000000000000000000000000'::bytea;
```

Comment:
Have you estimated time for this query? It looks like it will take a lot of time and doesn't differ much from executing full migration in SQL

Reply:
No, I haven't. What's the best way to do this? Would having nullable `fee_account_address` be better in terms of performance?

Reply:
You can test it on stage DB, i.e. just run
```
ALTER TABLE miniblocks
    ADD COLUMN fee_account_address BYTEA NOT NULL DEFAULT '\x0000000000000000000000000000000000000000'::bytea;
```
vs
```
ALTER TABLE miniblocks
    ADD COLUMN fee_account_address BYTEA;
```

and drop the column after the test

Reply:
Both `ALTER TABLE` statements execute almost instantly on the stage DB. Could there be differences in how they are handled down the line, or does this mean that the current approach is OK?

Reply:
I think it is OK then